priv/gettext/fr/LC_MESSAGES/privacy.po

```diff
@@ -14,175 +14,175 @@ msgstr ""
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:23
 #: lib/listudy_web/templates/page/privacy.html.eex:33
 msgid "User submitted information"
-msgstr ""
+msgstr "Informations soumises par l'utilisateur"
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:11
 #: lib/listudy_web/templates/page/privacy.html.eex:20
 msgid "What information Listudy collects"
-msgstr ""
+msgstr "Quelles informations sont collectées par Listudiy"
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:28
 msgid "When you visit Listudy, some information about the request is logged on the server. This information includes the time of the request, the HTTP method, the URI of the request, the status code generated by the server and the referrer of the request."
-msgstr ""
+msgstr "Lorsque vous visitez Listudy, certaines informations sur la demande sont enregistrées sur le serveur. Ces informations incluent l'heure de la requête, la méthode HTTP, l'URI de la requête, le code d'état généré par le serveur et le référent de la requête."
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:12
 msgid "Why this information is collected and how it's used"
-msgstr ""
+msgstr "Pourquoi ces informations sont collectées et comment sont-elles utilisées"
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:31
 msgid "Why this information is collected and how it’s used"
-msgstr ""
+msgstr "Pourquoi ces informations sont collectées et comment sont-elles utilisées"
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:47
 msgid "You can edit or delete your user provided informations at any time."
-msgstr ""
+msgstr "Vous pouvez modifier ou supprimer vos informations utilisateurs à tout moment."
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:56
 msgid "You may choose to close your account at any time, for any reason."
-msgstr ""
+msgstr "Vous pouvez choisir de fermer votre compte à tout moment, pour quelque raison que ce soit."
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:34
 msgid "Your email address will remain confidential and is used to log in or in case you forget your password. Moderators are able to set email addresses for accounts to assist in troubleshooting."
-msgstr ""
+msgstr "Votre adresse e-mail restera confidentielle et est utilisée pour vous connecter ou en cas d'oubli de votre mot de passe. Les modérateurs peuvent accéder aux adresses e-mail des comptes afin de faciliter le dépannage."
 
 #, elixir-format
 #: lib/listudy_web/templates/page/privacy.html.eex:40
 msgid "Your information will only be disclosed if it is legally necessary to do so. For example, if Lichess is ordered to hand over a certain user’s information, or if someone hacks Lichess."
```

Comment:
This seems like a copy/past leftover from Lichess. Should it be updated to Listudy?

Reply:
Yes its a copy/paste leftover. It shouldn't get fixed in the po file but in the original place. If you fix it you can use `scripts/gettext/merge.sh` (which requires you have elixir and mix installed) to update the po files. If thats not possible for you let me know and I will do it.

Reply:
I never used `elixir` before so I just installed it.
When I ran the script, a lot of other files  (`.po` and `.pot`) were modified. Should I commit them too?

And in each of the `privacy.po` file updated by this change, `#, elixir-format` was replace by `#, elixir-format, fuzzy` should I remove this `fuzzy`?

Reply:
No the po files have to be left unaltered other than in msgstr fields. Please commit the modifications done by the script too.